daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -219,6 +225,11 @@ object Node {
     def versionedKey: VersionedKeyWithMaintainers = versioned(key)
   }
 
+  final case class InterfaceKeyWithMaintainers(
+    interfaceId: TypeConName,
```

Comment:
For an `exerciseByInterfaceKey @I @J key`, is this the ID of the interface that defines the key (`J`) or of the interface we want the key to be downcast to (`I`)?